src/ipc.cpp

```diff
@@ -16,37 +16,39 @@
 
 
 #include "src/ipc.h"
+#include "src/misc/settings.h"
 #include <QDebug>
 #include <QCoreApplication>
+#include <unistd.h>
 
-IPC::IPC() :
-    globalMemory{"qtox"}
+
+IPC::IPC()
+    : globalMemory{"qtox-" GIT_VERSION}
```

Comment:
Is there any reason you would give a separate memory to each version ?

That seems bad, having multiple clients running on the same profile could cause inconsistencies, and after an update the new client obviously won't be able to talk to previous, which could be confusing.

If you just want to prevent clients with different IPC protocols from messing with each other, I'd suggest using a new fixed name for the global memory like "qtox2" or "qtox-#1243".